packages/rx_bloc_cli/lib/src/models/generator_arguments.dart

```diff
@@ -165,4 +165,12 @@ class GeneratorArguments
   bool get deepLinkEnabled => _showcaseConfiguration.deepLinkEnabled;
 
   /// endregion
+
+  /// region Other
+
+  /// Flag indicating if the project uses OTP functionality
+  bool get hasOtp =>
+      _authConfiguration.otpEnabled || _featureConfiguration.onboardingEnabled;
```

Comment:
Do we really need to create a dependency on the OTP ?

Reply:
Both, `OTP` and `Onboarding` features, use the `widget_toolkit_otp` dependency. This flag is set so that we properly generate the files. I was planning on going with `uses_widget_toolkit_otp` but this seemed too long for a name so I used `has_otp`. Let me know if you want it changed.